el/index.js

```diff
@@ -82,16 +82,66 @@ module.exports = yeoman.Base.extend({
 
     var el;
     var pathToEl;
+    var pathToElements;
+    var pathToBowerComponents;
+    var pathToApp;
+    
+    // Let you specify a path to the app folder
+    if (this.flags.app) {
+
+      // pathToApp = 'subfolder/app'
+      pathToApp = this.flags.app;
+
+      // adding / to the end of the path
+      if (pathToApp.charAt(pathToApp.length - 1) !== '/' ) {
```

Comment:
In the previous code it do also have the `/` hardcoded.
`    pathToEl = path.join(this.destinationPath('app/elements'), this.flags.path);` 
I have only moved the hardcoded values out to variables that can be changed with the flags.